src/SmartComponents/Inventory/EntityDetail.js

```diff
@@ -25,9 +36,90 @@ class EntityDetails extends Component {
             isOpen: collapsed
         });
     }
-    render() {
-        const { loaded, entity } = this.props;
+
+    generateTop = () => {
+        const { entity } = this.props;
         const { isOpen } = this.state;
+        return (
+            <Grid className="ins-entity-header">
+                <GridItem md={ 6 }>
+                    <Title size='2xl'>{ entity && entity.display_name }</Title>
+                </GridItem>
+                <GridItem md={ 6 }>
+                    <Dropdown
+                        onSelect={ this.onSelect }
+                        toggle={ <DropdownToggle onToggle={ this.toggleActions }>Actions</DropdownToggle> }
+                        isOpen={ isOpen }
+                        position={ DropdownPosition.right }
+                        dropdownItems={ [
+                            <DropdownItem key="1">Some action</DropdownItem>
+                        ] }
+                    />
+                </GridItem>
+            </Grid>
+        );
+    }
+
+    generateFacts = () => {
+        return (
+            <Grid className="ins-entity-facts">
+                <GridItem md={ 6 }>
+                    <div>
+                        <span>
```

Comment:
Should this be a` <Level>`?

Reply:
Not really Level will split these two into seperate items, but here we want to have them aligned next to each other.